wasm/derive/src/entrypoint.rs

```diff
@@ -74,6 +57,29 @@ impl super::params::ConstructArg for ParamType {
     }
 }
 
+fn infer_args(sign: &syn::Signature) -> Punctuated<syn::Expr, syn::token::Comma> {
+    let mut args = vec![];
```

Comment:
the same point as [here](https://github.com/hyperledger/iroha/pull/3299#discussion_r1132646776)